src/tir/schedule/primitive/blockize_tensorize.cc

```diff
@@ -565,58 +569,230 @@ StmtSRef Blockize(ScheduleState self, const StmtSRef& loop_sref, bool preserve_u
   return result;
 }
 
-BlockRealize BlockizeBlocks(const ScheduleState& self, const Array<StmtSRef>& block_srefs,
-                            const StmtSRef& lca, Map<Block, Block>* block_sref_reuse,
-                            bool preserve_unit_iters) {
+class CollectSubstInfo : public StmtVisitor {
```

Comment:
Please depict what this visitor do in comments.

Reply:
It is to collect the external loop information of the common ancestor of the block list, and to replace some block and loop variables in BlockizeBlocks

Reply:
Please add these comments to the class's code.

Reply:
done!

Reply:
done!